src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -328,17 +330,14 @@ def lsblk_all(device='', columns=None, abspath=False):
 
     result = []
 
+    if device:
+        return _lsblk_parser(' '.join(out))
```

Comment:
I don't understand. Can the output have multiple lines (if we use `--nodeps` as proposed above)? Otherwise `out` contains a single string, right?

Reply:
yes, it can have multiple lines.

`lsblk_all()` can be called without `device` being set.

Reply:
Ah, I have mistaken `if device` with `if not device`.

Reply:
Yes, but I mean in the case where `device` is set. You know, this line here is only executed in case `if device`.